imagersite/imager_images/templates/library.html

```diff
@@ -5,24 +5,34 @@
 <div class="container">
      {% for album in user.albums.all %}
         <div class="col-lg-3 col-md-4 col-sm-5 col-xs-6">
-        <h4 class='text-center'>{{ album.title }}</h4>
-        <a href='/images/albums/{{ album.id }}'><img style='box-shadow: 10px 15px 15px; width: auto; height: 160px;' class="thumbnail" src="{{ MEDIA_URL }}{{ album.cover.file }}"></a>
-        </div>
+        <h4 class='text-center'>{{ album.title }}</h4><a href="{% url 'edit_album' album.id %}">Edit</a>
+        <a href='{% url 'album' album.id %}'>
+
         {% if album.cover.file %}
-            src="{{ MEDIA_URL }}{{ album.cover.file }}">
+          <img style='box-shadow: 10px 15px 15px; width: auto; height: 160px;' class="thumbnail" src="{{ MEDIA_URL }}{{ album.cover.file }}">
         {% else %}
-            src="{{ MEDIA_URL }}seattle.jpg">
+          <img style='box-shadow: 10px 15px 15px; width: auto; height: 160px;' class="thumbnail" src="{{ MEDIA_URL }}seattle.jpg">
```

Comment:
I'd love to see the css for this pulled out of the tag and put in a css file.  Inline style declarations are hell to override.